segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/tables/IteratorArgs.java

```diff
@@ -24,9 +24,11 @@
 @Builder
 public class IteratorArgs {
     /**
+     * EXPERIMENTAL!
```

Comment:
To clarify, is it only the `prefixFilter` that is experimental? Basically, `getpPrefixFilter` returns an `ArrayView` that is not guaranteed to be correct? Do you know whether this is displayed correctly with javadocs (because of the @Data lombok annotation)?

Reply:
We do not generate or publish Javadocs for internal services since everything is behind the wire protocol, so this is just for someone looking at the code. 

My IDE correctly displays Javadoc when I hover over a method, so I am not worried about it here.

And yes, it's only the `prefixFilter` that is experimental, since the whole Sorted Table Segment is experimental too.

> getpPrefixFilter returns an ArrayView that is not guaranteed to be correct
No. Making use of this feature is experimental. There is nothing here that indicates something isn't going to be correct.